bigquery/api/sync_query.py

```diff
@@ -66,7 +66,7 @@ def main(project_id, query, timeout, num_retries):
             pageToken=page_token,
             **query_job['jobReference']).execute(num_retries=2)
 
-        print(json.dumps(page['rows']))
+        print(json.dumps(page.get('rows', [])))
```

Comment:
Ah! so you're saying paging would break the original script's JSON output?

This is a nice solution, good idea.

The valid JSON output is very nice in combination with jq because the raw output can be a bit.. raw :)

+1 !